### DISCUSSION TITLE
[A-L] (2023/24) Foglio 9 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#196] [Esercizi - Bertini]
![Screenshot 2023-11-27 134915](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/db0ed872-1f6e-4577-86af-ae562167c389)

**Comment by cornflexxx** [comment]
Riscrivo la formula del valore atteso:

$$\mathbb{E}(X)=\sum_{k=0}^{+\infty} k\cdot \mathbb{P}(X=k)=\sum_{k=1}^{+\infty} \sum_{i=1}^{k}\mathbb{P}(X=k)$$

per comodità d'ora in poi scriverò $\mathbb{P}(X=k)$ come $p(k)$.
Quindi svolgendo le sommatorie otteniamo la somma delle probabilità al variare di k:

$$\mathbb{E}(X)=p(1)+p(2)+p(2)+...+[p(n)+p(n)+...+p(n)]+... \quad (i) \qquad \text{ le quadre racchiudono esattamente $n$ termini}$$

Usando la _prop. associativa_ riscriviamo $(i)$ nel seguente modo:

$$\mathbb{E}(X)=(p(1)+p(2)+...+p(n)+...)+(p(2)+p(3)+...+p(n)+...)+...+(p(n)+p(n+1)+...)+...=...$$


$$...=\sum_{k=0}^{+\infty} \sum_{h=k+1}^{+\infty} \mathbb{P}(X=h)\quad (ii)$$

Ricordiamo che : 
$$\mathbb{P}(X>k) = \mathbb{P}(X=k+1)+\mathbb{P}(X=k+2)+... =\sum_{h=k+1}^{+\infty} \mathbb{P}(X=h) \qquad \text{fino al massimo valore assunto dalla v.a}$$  
Allora possiamo riscrivere così la $(ii)$:
$$\mathbb{E}(X)=\sum_{k=0}^{+\infty}\mathbb{P}(X>k)$$

**Comment by CiottoloMaggico** [comment]
@cornflexxx come hai fatto a far sparire il k nella prima sommatoria? kP(X=k) = P(X<=k)? 
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/aa1e4938-3cf1-45cf-87d6-1430d2829f78)

**Comment by cornflexxx** [comment]
> @cornflexxx come hai fatto a far sparire il k nella prima sommatoria? kP(X=k) = P(X<=k)?
> ![image](https://private-user-images.githubusercontent.com/79638284/287506115-aa1e4938-3cf1-45cf-87d6-1430d2829f78.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE1OTU1MjAsIm5iZiI6MTcwMTU5NTIyMCwicGF0aCI6Ii83OTYzODI4NC8yODc1MDYxMTUtYWExZTQ5MzgtM2NmMS00NWNmLTg3ZDYtMTQzMGQyODI5Zjc4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjAzVDA5MjAyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY4YjUxYjg2ZjQxN2Q2YzQ4MGUzNDA0ODQ0N2ZhYzc2MDQ0Njg4YmIzOGU1NTNhZGRiNWE0ZWVhZmEwY2QzYzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.iXi3P7EM4SJqqVgCr222FqFs1cj5Ojdv-a3xM6o31UA)

Non ho scritto quello perché se noti nella sommatoria si calcola sempre la distribuzione di $k$ non della $i$, ho solo osservato che svolgendo la sommatoria $p(k)$ compare $k$ volte

**Comment by CiottoloMaggico** [comment]
Non ci avevo fatto caso, sicuramente la tua soluzione è più semplice della mia. A farlo in quel modo non ci ho proprio pensato 🤯
